mobile-first.html

```diff
@@ -209,14 +209,43 @@ <h2>Skills</h2>
 
         
       </div>
-  
+      <div class="box5">
+        <h1 class="personalmessage"> I'm always<br>interested in hearing<br> about new projects,<br> so if you'd like to chat<br> please get in touch.
+          <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+            <div>
+              <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+            </div> 
+            <div>
+              <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
+              <textarea class="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+              <input class="send" type="submit" value="Get in touch">
+            </div>
+                      
+          </form>
+          
           
-             
+        
+
+      </div>    
+      <div class="box6">        
+          <hr>
+            <ul class="footer-container">
+              <li><a><img src="images/github.png" alt="github"/></a></li>
+              <li><a><img src="images/linkedin.png" alt="linkedin"/></a></li>
+              <li><a><img src="images/angel list.png" alt="angel-list"/></a></li>
+              <li><a><img src="images/twitter.png" alt="twitter"/></a></li>
+              <li><a><img src="images/medium.png" alt="medium"/></a></li>
+            </ul>
+            <div class="ending-line">
+            </div>
+        
+
+      </div>       
        
         
         
     </div>
-    <!-- navbar -->
```

Comment:
it is advisable to remove unnecessary comments in your code since they are classes or ids which can be used to name your attributes, otherwise great job so far, you put in a lot of hard work